### ISSUE TITLE
Charts cannot be accessed by screen reader users because there are no alternative options available: A11y_Visual Studio Code Jupyter Extensions_Home_Screenreader

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-jupyter#17077]
### GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #A11ySev2;#DesktopApp; #Visual Studio Code Jupyter Extensions; #GH-VSCodeJupyterExtensions-Win32-Oct2025; #WCAG1.3.1;#NVDA;#ScreenReader;#Info and Relationships;

### Actual Experience.
Charts cannot be accessed by screen reader users because there are no alternative options available

### Expected Experience
Charts information should be Fully accessible via screen reader

### Environment Details:
Application: Visual Studio Code Jupyter Extensions
Version: 1.105.0-insider (user setup)
OS build :26100.4770

### Repro Steps

1. Install "GitHub Copilot app modernization - upgrade for Java" app.
2. Launch the Visual studio code.
3. "Chat" popup will appear. Type "upgrade this project to java 21 using java upgrade tools" in edit field and hit ENTER key.
4. Verify that Charts cannot be accessed by screen reader users because there are no alternative options available

### User Impact:
Screen reader users are unable to perceive or interact with chart content, which significantly impairs their ability to understand data visualizations. This leads to: Exclusion from critical insights conveyed through charts and graphs.

### Attachment

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/7c063bfd-a780-45aa-8d1c-c42ca13eaf50" />
<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/5f79fc72-9cd3-40bb-93ff-4a35ab136d86" />
